src/test/java/core/basesyntax/service/TransactionParserImplTest.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.impl.TransactionParserImpl;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+
+public class TransactionParserImplTest {
+    private static TransactionParser transactionParser;
+    private static List<String> transaktions =
+            List.of("type,fruit,quantity", "b,apple,10", "s,orange,20");
+    private static FruitTransaction transaction1 =
+            new FruitTransaction(FruitTransaction.Operation.BALANCE, "apple", 10);
+    private static FruitTransaction transaction2 =
+            new FruitTransaction(FruitTransaction.Operation.SUPPLY, "orange", 20);
+
```

Comment:
```suggestion
    private static FruitTransaction firstTransaction/appleTransaction =
            new FruitTransaction(FruitTransaction.Operation.BALANCE, "apple", 10);

```

It is bad practice to use numbers in variable names